packages/framework/framework/src/subscribers/subscriber-loader.ts

```diff
@@ -213,20 +216,20 @@ export class SubscriberLoader {
   }
 
   async load() {
-    let hasSubscriberDir = false
-
-    try {
-      await access(this.#sourceDir)
-      hasSubscriberDir = true
-    } catch (err) {
-      logger.debug(`No subscriber directory found in ${this.#sourceDir}`)
-    }
+    const normalizeSourcePaths = Array.isArray(this.#sourceDir)
+      ? this.#sourceDir
+      : [this.#sourceDir]
+    const promises = normalizeSourcePaths.map(async (sourcePath) => {
+      try {
+        await access(sourcePath)
+      } catch {
```

Comment:
Sugg: let's throw here instead? Much better experience knowing something explicitly screwed up than it getting swallowed

Reply:
It is more if you have a plugin that does not provide any subscribers, then the dir wont exists but it is expected. It would throw mater though if we cant properly load a subscriber

Reply:
So i believe it would create more confusion to throw at this level, probably a user would wonder why it throw that it cant load the subscriber while I don't provide any subscribers and don't want to provide any

Reply:
eventually I can add info log to say that it will be skipped for that source? wdyt?

Reply:
I thought this config was exposed to the user, looks like its not. You're right, this doesn't require a throw. 🤞🏻

Reply:
I think adding the log is sufficient, WDYT Riqwan?

Reply:
I ve added them, let me know what you think guys